content_test_filtering.py

```diff
@@ -55,11 +59,19 @@
         already_analysed.append(file_record["filepath"])
         # If change affected any other file -> analyse it
         changed_files.extend(diff_structure.affected_files)
-
     list_of_tests = connect_to_labels.get_labels(tests, options.output)
     if options.output == "json":
         logs.print_json(list_of_tests)
     else:
         logs.print_all_logs(list_of_tests, output_format=options.output_format)
-
+    # Save the updated controls to a file for syncing OSCAL catalog
+    logger.debug(f"The updated controls: {controls_files}")
+    if options.output == "json":
```

Comment:
There is already the same condition above that could be used.

Reply:
Thanks. Updated.